src/hmm/tree-accu.cc

```diff
@@ -84,8 +84,7 @@ void AccumulateTreeStats(const Transitions &trans_model,
       for (int32 j = 0; j < static_cast<int32>(split_alignment[i+info.central_position].size());j++) {
         // for central phone of this window...
         EventType evec_more(evec);
-        int32 pdf_class = trans_model.TransitionIdToPdfClass(
-            split_alignment[i+info.central_position][j]);
+        int32 pdf_class = trans_model.PdfClassForTid(split_alignment[i+info.central_position][j]);
```

Comment:
Unrelated to this PR: The whole build-tree code uses 0-based integers for pdf-classes but the new topology code uses 1-based numbers. I think we should change the tree codes (e.g. in BuildTree, GetStubMap, and GenRandStats). @danpovey, any comments?

Reply:
Oh, you're right; whenever pdf-classes appear in scripts or in defaults, they will have to be changed (e.g. I think the default for clustering the phones when using triphone models is just to use pdf-class 1, but that should be changed to be 2).

Reply:
I'll make a PR for it.